elifetools/parseJATS.py

```diff
@@ -3382,7 +3384,7 @@ def references_json(soup, html_flag=True):
             set_if_value(ref_content, "dataId", ref.get("accession"))
 
         # doi
-        if ref.get("publication-type") not in ["web"]:
+        if ref.get("publication-type") not in ["web", "webpage"]:
```

Comment:
Could maybe move to a constant as used in multiple places now and has multiple items? e.g.
`WEB_PUBLICATION_TYPES = ["web", "webpage"]`

Could do the same for the variations, though depends on usage.

Reply:
Maybe, although because of the varied rules for each type of reference and the data to be specified - either required by the JSON schema, or to set JSON values - the types of reference are not always grouped together. If you search for all the places the new type ``"webpage"`` is used, it is part of these conditionals

```
== "webpage"
in ["web", "webpage"]
in ["patent", "web", "webpage"]
in ["confproc", "data", "web", "webpage", "preprint"]
```

Messy as they look, I think each reference type is probably clearer to understand when listed directly in the code, rather than trying to express them into constants, even though ``"web"`` and ``"website"`` are generally used at the same time (except in the first example) and that pair might be suitable to be a constant as you suggest.